build.sbt

```diff
@@ -12,28 +12,29 @@ lazy val taxYear = (project in file("."))
       )
     )
   )
-  .settings(majorVersion := 3)
+  .settings(majorVersion := 4)
   .settings(isPublicArtefact := true)
   .settings(ScoverageSettings())
   .settings(
-    resolvers += Resolver.typesafeRepo("releases")
+    resolvers += Resolver.typesafeRepo("releases"),
+    scalafmtOnCompile := true
   )
 
 libraryDependencies ++= Seq(
   "org.threeten"         % "threeten-extra" % "1.7.2",
-  "com.vladsch.flexmark" % "flexmark-all"   % "0.36.8"  % Test,
-  "org.pegdown"          %  "pegdown"       % "1.6.0"   % Test
+  "com.vladsch.flexmark" % "flexmark-all"   % "0.36.8" % Test,
+  "org.pegdown"          % "pegdown"        % "1.6.0"  % Test
 )
 
 libraryDependencies := {
   CrossVersion.partialVersion(scalaVersion.value) match {
     case Some((2, scalaMajor)) if scalaMajor <= 11 =>
       libraryDependencies.value ++ Seq(
-        "org.scalatestplus"      %% "scalatestplus-scalacheck"  % "3.1.0.0-RC2" % Test
+        "org.scalatestplus" %% "scalatestplus-scalacheck" % "3.1.0.0-RC2" % Test
```

Comment:
Not sure we should be using RC's here, can we go back to the last stable version?

Reply:
Ooh, that's a good point. Didn't consider it. According to the [official Scalatest docs](https://www.scalatest.org/user_guide/property_based_testing), that appears to be `scalacheck-1-17`. However, [this is only available from Scala 2.12](https://central.sonatype.com/search?q=scalacheck-1-17). For Scala 2.11, [it appears you have to use Scala Check 1.15](https://central.sonatype.com/artifact/org.scalatestplus/scalacheck-1-15_2.11).